### ISSUE TITLE
Android 15, editing window remains behind keyboard, and upper menu behind status line

### ISSUE THREAD

**Issue by MartinuDuo** [gsantner/markor#2451]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**. Not a [question or feature request](https://github.com/gsantner/markor/issues/new/choose).
- [X] The topic is **not** already reported at [Issues](https://github.com/gsantner/markor/issues?q=#js-issues-search). _(I've searched it)_.
- [X] Markor **is** up to date. See [Releases](https://github.com/gsantner/markor/tags) for the latest version. Updates are available from [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/) and GitHub.
- [X] The bug is still present in the latest development version (git master). (Please [download](https://nightly.link/gsantner/markor/workflows/build-android-project/master) and try the test version of Markor, named **Marder**. Don't worry; Markor and Marder appear as completely separate applications. You can install both side-by-side, and Markor settings are not touched. In case the issue is resolved there, you don't need to create a bug report. The change will be part of the next Markor update.)

### Description

When editing a .MD document on android 15, pixel 8 pro. The editing pane remains behind the keyboard. As you type a long document, you end up typing behind the keyboard. I would expect the editing pane to be fully above the keyboard, as per previous versions, so you can see what you are editing. 
Also the upper menu now resides behind the status line, so nothing can be selected.

This did not occur before the update to Android 15

### Steps to reproduce

1. Open new document
2. Notice that the upper menu is behind the top status line

![Screenshot_20241018-122835](https://github.com/user-attachments/assets/c9afdbb7-1ecc-41bb-a925-afbf2825c37a)

![Screenshot_20241018-122737](https://github.com/user-attachments/assets/700bd821-33a5-4548-982d-3d232e9e5cc9)

Type lots of lines till the new lines are appearing behind the keyboard, you can see the top of the red editing caret in the second screenshot
4. 
5. 

### Information

Android version: 15
Device: Pixel 8 Pro
App Version: Latest (upper menu is unselectable)

### Source

Google Play

### Format / File type

Not specific

**Comment by gsantner** [comment]
>   The topic is not already reported at Issues. (I've searched it).

Why you do you tick the checkbox and ignore what it says.

**Comment by MartinuDuo** [comment]
> > The topic is not already reported at Issues. (I've searched it).
> 
> Why you do you tick the checkbox and don't care what it says?

Why do you assume I didn't do a search and didn't find it. If it already exists, then please provide link. I took the time and effort to report this issue,. Your response is rude

**Comment by gsantner** [comment]
#2448 #2387

**Comment by gsantner** [comment]
Sorry but it should even say issues are closed without notice when people dont try to follow the steps.

Our maintainers time is limited too. Reading through issues takes time too, and no, just by writing here nobody is suddenly entitled to any particular way of reply. 

It is no rude to close duplicates.

**Comment by MartinuDuo** [comment]
I've no problem with closing for duplicates, and I fully appreciate the
time it takes.

It's not rude to say "Closed - duplicate"

It is rude to infer I just clicked a box, didn't care what it said, and
wasted your time for lack of my own.

I took the time to look for app support, read, login, search, describe the
issue, screenshot, upload - about 20-30 min in all.

I made the effort to what I hoped was contributing and helping, but your
snide reply after I did all this, made me wish I'd not wasted time, your
time, and just left a 1 star review saying "Doesn't work with Android 15".

Just say "closed - duplicate" in future, and you won't upset anyone.
